public/app/features/alerting/unified/api/receiversApi.ts

```diff
@@ -6,6 +6,8 @@ import { CONTACT_POINTS_STATE_INTERVAL_MS } from '../utils/constants';
 
 import { alertingApi } from './alertingApi';
 import { fetchContactPointsState } from './grafana';
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { notificationsAlertingV0alpha1Endpoints } from '@grafana/hackathon-13-registrar-private/rtk-query';
```

Comment:
I'm getting lots of type errors if I try to generate hooks, e.g. `TS2339: Property useUpdateTimeIntervalMutation does not exist on type`, I generated the endpoints via `npm run generate` in `generated/typescript/private`, do I need to do anything else or are the hooks not supported?

Reply:
RTK client is already generated in Registrar, you don't need to do it yourself. And hooks are there when you create Api and inject endpoints. See below:

https://github.com/user-attachments/assets/bee5a95f-8824-4dcb-805c-2f7b97157e2e



Happy to huddle if you want so we go through it together